dependencies/CMakeLists.txt

```diff
@@ -15,3 +15,19 @@ ExternalProject_Add(libmodbus
         ${LIBMODBUS_ADDITIONAL_ARGS}
         INSTALL_DIR ${CMAKE_PREFIX_PATH}
         )
+
+# And add WolkSDK-Cpp if there is no WolkSDK-Cpp
+if (NOT TARGET WolkAboutCoreUtility)
+    ExternalProject_Add(WolkSDK-Cpp
+            GIT_REPOSITORY "https://github.com/Wolkabout/WolkSDK-Cpp"
+            GIT_TAG "cc3f9611cb02355815a9944c45f05fe8b4c3e4d8"
```

Comment:
We should make tag on sdk for 22 GA and target that tag

Reply:
As soon as this gets pulled in, I can make one.
https://github.com/Wolkabout/WolkSDK-Cpp/pull/56